system_tests/test_happybase.py

```diff
@@ -877,3 +879,20 @@ def test_counter_dec(self):
         self.assertEqual(
             table.row(ROW_KEY1, columns=[COL1]), {COL1: _PACK_I64(-dec_value)}
         )
+
+
+class TestTable_region(BaseTableTest):
+
+    def test_region(self):
+        ALT_TABLE_NAME = "table_with_split_key"
+        connection = Config.CONNECTION
+
+        connection.create_table(ALT_TABLE_NAME, {COL_FAM1: {}})
+        table = connection.table(ALT_TABLE_NAME)
```

Comment:
Please create a `google-cloud-python` `Table` with `initial_split_keys`, so that you can get meaningful `regions` from the HappyBase `table.regions()` call.

Reply:
added system test with passed split key in table